File: BudgetBringer/src/models/responses/process/process-approved/response-process-approved-summary.ts
import {ResponseProcessApprovedSummaryDetail} from "./response-process-approved-summary-detail";

/**
 * 결과중 개별 승인 별 통계 데이터 모음
 */
export class ResponseProcessApprovedSummary {
  /**
   * 오너정보
   */
  items: ResponseProcessApprovedSummaryDetail[];
}

File: Models/Responses/Process/ProcessApproved/ResponseProcessApprovedSummary.cs
namespace Models.Responses.Process.ProcessApproved;

/// <summary>
/// 결과중 개별 승인 별 통계 데이터 모음
/// </summary>
public class ResponseProcessApprovedSummary
{
    /// <summary>
    /// 오너정보
    /// </summary>
    // ReSharper disable once CollectionNeverQueried.Global
    public List<ResponseProcessSummaryDetail<ResponseProcessApproved>> Items { get; set; } = [];
}